README.md

```diff
@@ -428,6 +428,31 @@ machines, see the ["QEMU and Kubernetes"](#qemu-and-kubernetes) section below.
     $ kubectl label node <your node> storage=pmem
 ```
 
+- **Set up certificates**
+
+Certificates are required as explained in [Security](#security).
+If you are not using the test cluster described in
+[Starting and stopping a test cluster](#starting-and-stopping-a-test-cluster)
+where certificates are created automatically, you must set up certificates by
+running the `./test/setup-ca-kubernetes.sh` script manually in your cluster.
+This script requires "cfssl" tools which can be downloaded.
+These are steps for manual set-up of certificates:
+
+- Download cfssl tools
+
+  Alternatively, one can run three commands below via make _work/.setupcfssl-stamp.
```

Comment:
Now the "alternatively" makes no sense anymore. Let's remove the sentence completely. We should just show the recommended way of doing this, mentioning alternatives just makes the reader wonder which one they should pick.

The rationale behind showing the individual commands is:
- no need for make
- users can see what they need to do if they don't have curl or want to use something else

Reply:
> Let's remove the sentence completely.

removed it.